samples/Basic YARP Sample/appsettings.json

```
@@ -0,0 +1,56 @@
+{
+  // Base URLs the server listens on, must be configured independently of the routes below
+  "Urls": [
+    "http://localhost:5000",
+    "https://localhost:5001"
+  ],
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      // Uncomment to hide diagnostic messages
+      // "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "ReverseProxy": {
+    // Defines the routes the proxy will listen on
```

This is ambiguous with the urls given above.
```suggestion
    // Routes tell the proxy which requests to forward.
```